refinery_core/src/traits/sync.rs

```diff
@@ -132,6 +113,11 @@ where
         ASSERT_MIGRATIONS_TABLE_QUERY.replace("%MIGRATION_TABLE_NAME%", migration_table_name)
     }
 
+    fn assert_migrations_table(&mut self, migration_table_name: &str) -> Result<usize, Error> {
+        self.execute(&[&Self::assert_migrations_table_query(migration_table_name)])
```

Comment:
is `assert_migrations_table_query` being used elsewhere? If not can we inline it here?

Reply:
yup, np